src/plugins/audioplayers/player.c

```diff
@@ -0,0 +1,413 @@
+#include "gst/gst.h"
+#include "gst/gstelementfactory.h"
+#include "gst/gstmessage.h"
+#include "gst/gstsegment.h"
+#include "platformchannel.h"
+#include <flutter-pi.h>
+#include <plugins/audioplayers.h>
+#include <stdio.h>
+
+FILE_DESCR("AudioPlayer::player")
+
+// Private Class functions
+void AudioPlayer_SourceSetup(GstElement* playbin, GstElement* source, GstElement** p_src);
+gboolean AudioPlayer_OnBusMessage(GstBus* bus, GstMessage* message, AudioPlayer* data);
+gboolean AudioPlayer_OnRefresh(AudioPlayer* data);
+
+// Private methods
+void AudioPlayer_SetPlayback(AudioPlayer* self, int64_t seekTo, double rate);
+void AudioPlayer_OnMediaError(AudioPlayer* self, GError* error, gchar* debug);
+void AudioPlayer_OnMediaStateChange(AudioPlayer* self, GstObject* src, GstState* old_state, GstState* new_state);
+void AudioPlayer_OnPositionUpdate(AudioPlayer* self);
+void AudioPlayer_OnDurationUpdate(AudioPlayer* self);
+void AudioPlayer_OnSeekCompleted(AudioPlayer* self);
+void AudioPlayer_OnPlaybackEnded(AudioPlayer* self);
```

Comment:
```suggestion
// Private Class functions
static void AudioPlayer_SourceSetup(GstElement* playbin, GstElement* source, GstElement** p_src);
static gboolean AudioPlayer_OnBusMessage(GstBus* bus, GstMessage* message, AudioPlayer* data);
static gboolean AudioPlayer_OnRefresh(AudioPlayer* data);

// Private methods
static void AudioPlayer_SetPlayback(AudioPlayer* self, int64_t seekTo, double rate);
static void AudioPlayer_OnMediaError(AudioPlayer* self, GError* error, gchar* debug);
static void AudioPlayer_OnMediaStateChange(AudioPlayer* self, GstObject* src, GstState* old_state, GstState* new_state);
static void AudioPlayer_OnPositionUpdate(AudioPlayer* self);
static void AudioPlayer_OnDurationUpdate(AudioPlayer* self);
static void AudioPlayer_OnSeekCompleted(AudioPlayer* self);
static void AudioPlayer_OnPlaybackEnded(AudioPlayer* self);
```
(and `lower_camel_case`)

anything that's only needed in this file can be `static`